packages/design-system/package.json

```diff
@@ -53,8 +53,8 @@
   ],
   "dependencies": {
     "@popperjs/core": "^2.4.4",
-    "@types/react": "^17.0.10",
-    "@types/react-dom": "^17.0.10",
+    "@types/react": "^18.0.37",
+    "@types/react-dom": "^18.0.11",
```

Comment:
For some reason I ran into TS errors running `yarn build` (not `yarn build:storybook`). Errors were confined to Autocomplete (for Downshift types), StepList and Toggle (for React focus types I believe)